Calendar/app/src/main/java/com/elysium/calendar/CursorAdapter.java

```diff
@@ -0,0 +1,85 @@
+package com.elysium.calendar;
+
+import android.content.ContentUris;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.database.Cursor;
+import android.icu.util.Calendar;
+import android.os.Build;
+import android.provider.CalendarContract;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+/**
+ * Created by jay on 11/29/16.
+ */
+
+public class CursorAdapter extends android.support.v4.widget.CursorAdapter {
```

Comment:
Your class needs to extend `RecyclerView.Adapter`, not `CursorAdapter`. `CursorAdapter` does not work with RecyclerViews.